roboteam_ai/src/control/ballHandling/RotateAroundBall.cpp

```diff
@@ -10,11 +12,10 @@ namespace rtt {
 namespace ai {
 namespace control {
 
-RobotCommand RotateAroundBall::getRobotCommand(const world::Robot::RobotPtr &r, const Vector2 &targetP,
+RobotCommand RotateAroundBall::getRobotCommand(const world::Robot::RobotPtr &robot, const Vector2 &targetP,
```

Comment:
If you are going to remove const ref from getRobotCommand do it everywhere please.

Reply:
fixed